css/style.css

```diff
@@ -0,0 +1,73 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+/* body{
+    display: block;
+} */
+
+#navbar{
+    /* border: 2px solid red; */
+    position: fixed;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    min-width: 250px;
+    overflow: auto;
+    border-right: 2px solid black;
+    background-color: white;
+}
+
+header{
+    padding: 0.5em 0 0.5em 1em;
+}
+
+.nav-link{
+    text-decoration: none;
+    color: black;
+    text-align: left;
+    border-top: 2px solid black;
+    padding: 0.5em 0 0.5em 1em;
+    margin-top: 0.5em;
+}
+
+main{
+    margin-left: 260px;
+}
+
+ul{
+    list-style: outside;
+    padding-left: 3em;
+}
+
+section{
+    padding-top: 1em;
+}
+
+code{
+    display: block;
+    background-color: rgb(202, 202, 202);
+    padding: 0.3em 0.5em 0.3em 1em;
+    margin: 0.5em;
+    white-space: pre-line;
+    max-width: 55em;
+}
+
+
+@media screen and (max-width:600px){

```

Comment:
- [ ] Please be more polite, professional, friendly and transparent in your comments while doing code reviews. Your comment looks like you are ordering and do have not a strong reason to ask for adding something to the document. Kindly also make sure that your comments don't have any grammatical mistakes, please use *Grammarly* for your grammar check.

## Your comment

> When we open mobile view at browser, we cannot see the responsive effect. Maybe you should build the mobile view first then desktop later.